src/index.html

```diff
@@ -4,9 +4,43 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+
+    <main class="calendar calendar--month-length--29 calendar--start-day--thu">
+
+      <div class="calendar__cell calendar__cell--1 "></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
модифікатори в цьому завданні тільки для контейнера календаря мають застосовуватись, для днів крще використати `nth-child`